### PR TITLE
fits edits

### PR DESCRIPTION
**Pull request by rumachan**
Changes to notebook and README

### CODE REVIEW

File: FITS/FITS_data_access.ipynb [hunk 1]
```diff
@@ -826,12 +556,12 @@
     "    #make a temprary dataframe from the dictionary\n",
```
Comment (Thomas-Benson) [inline FITS/FITS_data_access.ipynb:556]: Typo: temprary to temporary

File: FITS/FITS_data_access.ipynb [hunk 2]
```diff
@@ -1113,12 +771,12 @@
...
```
Comment (Thomas-Benson) [inline FITS/FITS_data_access.ipynb:771]: Typo: temprary to temporary

File: FITS/FITS_data_access.ipynb [hunk 3]
```diff
@@ -730,12 +460,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You can use the FITS API to find sites in a particular area. https://fits.geonet.org.nz/api-docs/endpoint/site We use a ploygon around Ruapehu."
+    "You can use the FITS API to find sites in a particular area. https://fits.geonet.org.nz/api-docs/endpoint/site We use a ploygon around Tair&#259;whiti (East Cape). The kind of data present in FITS is not returned, but in this case all sites provide GNSS daily position solutions."
```
Comment (Thomas-Benson) [inline FITS/FITS_data_access.ipynb:463]: Typo: ploygon to polygon

**Review by Thomas-Benson — COMMENTED** [review]
Looks good, just a few minor typos that could be fixed

**Comment by rumachan** [comment]
Those changes made. My typing is shocking! Please re-review.

**Review by Thomas-Benson — APPROVED** [review]
I think some of those typos were there before your edits, they're just minors things I caught looking through it. I probably should have caught them myself when I did all the other ones. Either way the version of this tutorial on the J drive looks fine now, so you should be good to upload it to github.

**Pull request merged**